spec/controllers/bids_controller_spec.rb

```diff
@@ -155,7 +155,7 @@
           expect(flash[:bid]).to eq("success")
           expect(response).to redirect_to("/auctions/#{auction.id}")
 
-          bid = auction.bids.order('created_at DESC').first
+          bid = auction.bids.last
```

Comment:
I actually would like to make this a method or something so that if bids is reordered later it won't break